src/services/terminals.service.ts

```diff
@@ -47,6 +47,15 @@ class TerminalsService extends Service {
       .then((res) => Promise.resolve(options?.rawResponse ? res : new Terminal(res.data)));
   }
 
+  closeTerminal(id: number): Promise<number>;
+  closeTerminal(id: number, options: { rawResponse: true } & Options): Promise<AxiosResponse<any, any>>;
```

Comment:
Leftover `any`? Think this should be
```suggestion
  closeTerminal(id: number, options: { rawResponse: true } & Options): Promise<AxiosResponse<number, any>>;
```

Reply:
good spot